filesystem/FileMigrationHelper.php

```diff
@@ -35,7 +35,7 @@ public function run($base = null) {
 		// Loop over all files
 		$count = 0;
 		$originalState = \Versioned::get_reading_mode();
```

Comment:
We're not in a namespace, wonder why `\Versioned` used here. Would prefer `use \Versioned` anyway, makes it easier to refactor later. (Aware this patch doesn't introduce this oddity).